js_task2/src/js/index.js

```diff
@@ -0,0 +1,22 @@
+import '../scss/main.scss'
+import { getUsers } from './dataFunctions.js'
+import { selectWholeTable } from './helpers'
+import { baseUrl, addBtn, mainCheckbox } from './_variables.js'
+import { addUser } from './eventFunctions'
+
+window.addEventListener('DOMContentLoaded', () => {
+  getUsers(baseUrl)
+})
+
+addBtn.addEventListener('click', (Event) => {
+  Event.preventDefault()
+  addUser()
+})
+
+mainCheckbox.addEventListener('click', () => {
+  if (mainCheckbox.checked) {
+    selectWholeTable(true)
```

Comment:
when the main checkbox is checked there should be a possibility to delete all rows not by clicking some of the trash icons. but please show the delete button near the main row
![image](https://user-images.githubusercontent.com/84564899/126216456-b18a68f2-8909-4936-81f2-9d5c2fce5e50.png)

Reply:
done